src/h5p-helpers/render.js

```diff
@@ -79,11 +83,17 @@ exports.render = editor => {
                                     </div>
                                 </div>                                
                             </div>`
-                  )
-                  .join("")}
+        )
+        .join("")}
                 </div>
             </div>
-        </body>
+            <script>
+            function copyText(text)
+            {
+                navigator.clipboard.writeText(text);
```

Comment:
Yes that is correct it returns a Promise.

But since we don't have a feedback to show on UI (for now), I don't see a point writing it as .then ( {} ) .catch ( {} ). If you wish we could console log the error.

I was hoping to add feedback feature on the UI itself. But I keeping this feature for the next version after discussing with Thomas.

Reply:
Okay, if there is some kind of user feedback coming up I'm fine with it the way it is. I remember the `writeText` can fail for e.g. missing permissions and the user would be left in the dark why the text was not copied.